app/views/people/show.html.erb

```diff
@@ -0,0 +1,4 @@
+<h1>People#show</h1>
+<h1><%= @person.name %>'s ToDo:</h1>
+<div class = "individual-todo">
+  <h2><%=@person.tasks.where({completed: !=nil})%></h2>
```

Comment:
Try to keep your logic and data processing in the controller or model as much as possible, and use the view for displaying that information.